app/controllers/home_controller.rb

```diff
@@ -0,0 +1,38 @@
+class HomeController < ApplicationController
+  def index
+    @about = 'The ocean covers 71% of the Earth’s surface, and is host to an estimated 50 to 80% of all life on Earth. The ocean contains some of the most productive ecosystems, vast natural resources and unique habitats, and plays a vital role in regulating the Earth’s climate. However, the marine environment is facing many pressures, from the impacts of climate change to overfishing, microplastics and marine litter. Data and information are therefore required to support adaptive conservation and resource management that can help to mitigate these impacts, and to conserve the species and habitats that are necessary for our wellbeing. This metadata library addresses this lack of information by providing an overview of global marine and coastal datasets of biodiversity importance, including some datasets of regional interest. First established as a report in 2014, this library has grown quickly each year, with an ever-expanding list of ocean-related resources that can be found online, providing information on and links to these resources.';
+    
+    @themes = [
```

Comment:
Are these meant to be extracted from the database in the future?
My understanding is those are used to add content and style in the page; if so, can we use an helper for these as well?